packages/ui/src/components/LoadingLayer/index.module.scss

```diff
@@ -0,0 +1,33 @@
+@use '@/scss/underscore' as _;
+
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: var(--color-loading-layer);
+  @include _.flex-column;
+}
+
+.container {
+  width: 60px;
+  height: 60px;
+  border-radius: 8px;
+  background-color: var(--color-dark-background);
+  @include _.flex-column;
+}
+
+.loadingIcon {
+  animation: rotating 1.5s ease-in-out infinite;
```

Comment:
use `linear`?

Reply:
Keep it for now, I noticed most of the loading spin effects has a pausing effect more like ease-in-out. Anyway will check the final animation solution with Eric.

Reply:
Yeah, I see. np